.gitattributes

```diff
@@ -0,0 +1,2 @@
+test/testProject/**/* linguist-generated=true
```

Comment:
I *think* this should be `linguist-vendored` since it's not generated code, but just code that we didn't write. From https://github.com/github/linguist/blob/master/docs/overrides.md#vendored-code:
> Checking code you didn't write, such as JavaScript libraries, into your git repo is a common practice, but this often inflates your project's language stats and may even cause your project to be labeled as another language.

Reply:
Well it's up to whether we want these files to show up in diffs by default or not. I thought it'd be nice if these weren't in the diffs, what do you think?

Reply:
I'd assume that if we're updating the test projects it would be in a separate PR so those diffs wouldn't really clutter things up. But I could see them being annoying otherwise. It's up to you!